models/evms/evms_blocks.sql

```diff
@@ -0,0 +1,143 @@
+{{ config(
+        alias ='blocks',
+        unique_key=['blockchain', 'number'],
```

Comment:
if assigning unique keys here, do you want to apply the tests in schema file? for views, it's not vital, but this particular config setting won't do anything as-is (unless its more of a documentation kinda thing)

Reply:
again, same applies throughout all spells here